File: erezervisi_api/eRezervisi/eRezervisi.Common.Dtos/Reservation/ReservationUpdateDto.cs
namespace eRezervisi.Common.Dtos.Reservation
{
    public class ReservationUpdateDto
    {
        public int NumberOfAdults { get; set; }
        public int NumberOfChildren { get; set; }
        public DateTime From { get; set; }
        public DateTime To { get; set; }
        public string? Note { get; set; }
    }
}

File: erezervisi_desktop/erezervisi_desktop/lib/models/requests/reservation/reservation_update_dto.dart
class ReservationUpdateDto {
  late num numberOfAdults;
  late num numberOfChildren;
  late DateTime from;
  late DateTime to;
  late String? note;

  ReservationUpdateDto({
    required this.numberOfAdults,
    required this.numberOfChildren,
    required this.from,
    required this.to,
    this.note,
  });

  Map<String, dynamic> toJson() {
    return {
      'numberOfAdults': numberOfAdults,
      'numberOfChildren': numberOfChildren,
      'from': from,
      'to': to,
      'note': note,
    };
  }
}
